.pre-commit-config.yaml

```diff
@@ -11,6 +11,14 @@ repos:
     entry: mypy
     language: system
     types: [python]
+  - id: black
+    name: black
+    description: "Black: The uncompromising Python code formatter"
+    entry: black --line-length=100
```

Comment:
Can we set the line-length=100 in a `pyproject.toml`? So it's also used if someone uses `black .` on the command line, or the vscode plugin to run black automatically.

Reply:
Sorry I meant to have the discussion about the setup in #99. I'll implement these requests there

Reply:
Sure! But if we add it to the ci/cd, you'll have to reformat all files as well :) So I think it would make sense to have a single PR.